tests/System.UnitTests/TimedScopes/TimedScopeInstanceNameUnitTest.cs

```diff
@@ -0,0 +1,263 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using Xunit;
+using Xunit.Abstractions;
+using Microsoft.Omex.System.UnitTests.Shared;
+using Microsoft.Omex.System.TimedScopes;
+
+namespace Microsoft.Omex.System.UnitTests.TimedScopes
+{
+	/// <summary>
+	/// Unit tests for verifying functionality of TimedScopeInstanceName class
+	/// </summary>
+	[Trait("TestCategory", "SharedInfrastructure")]
```

Comment:
Not sure that this category makes sense anymore, @dakanev-msft do we still need it or maybe we should at least change category name?

Reply:
The other test projects in this repo don't have this trait so should be safe to remove it completely.